Sources/Testing/ABI/EntryPoints/EntryPoint.swift

```diff
@@ -396,8 +396,9 @@ func parseCommandLineArguments(from args: [String]) throws -> __CommandLineArgum
   }
 
   // Attachment output
-  if let attachmentsPathIndex = args.firstIndex(of: "--experimental-attachments-path"), !isLastArgument(at: attachmentsPathIndex) {
-    result.experimentalAttachmentsPath = args[args.index(after: attachmentsPathIndex)]
+  if let attachmentsPathIndex = args.firstIndex(of: "--attachments-path") ?? args.firstIndex(of: "--experimental-attachments-path"),
+     !isLastArgument(at: attachmentsPathIndex) {
```

Comment:
Is this "last argument" check new in this PR? Could we report an error if the flag is passed but it lacks a path afterward? Or am I misreading and it will produce an error already? I just don't want it to be silently ignored

Reply:
It's not new and serves as a bounds check so we don't read past the end of the array if `--attachments-path` is somehow the last argument.

I want to do a cleanup pass on the logic here to make it more legible and also add support for `--x=y` but it isn't a top priority.